js/containers/App.jsx

```diff
@@ -85,23 +86,27 @@ function App(props) {
 	);
 
 	return (
-		<Modal closeHandler="" title="Deployment">
-			<div className="row">
-				<div className="col-md-12">
-					<h3>Deployment options for ...</h3>
-				</div>
-				<div className="col-md-3">
-					<StepMenu
-						steps={steps}
-						value={props.activeStep}
-						onClick={props.onTabClick}
-					/>
-				</div>
-				<div className="col-md-9">
-					{content}
+		<div>
+			<NewDeployPlan />
+
+			<Modal show={props.isOpen} closeHandler="" title="Deployment">
```

Comment:
closeHandler? that should be set in the `mapDispatchToProps` so if the modal closes itself it can set the navigation.state = false.